### DISCUSSION TITLE
Where is the RMSE metric

### DISCUSSION THREAD

**Discussion by Boltzmachine** [Lightning-AI/torchmetrics#2708] [Regression] [answered]
I can only find `torchmetrics.image.RootMeanSquaredErrorUsingSlidingWindow`. Is it different from RMSE. if it is, where is the RMSE metrtic.

**Comment by Borda** [comment, marked as answer]
I think you are looking for https://lightning.ai/docs/torchmetrics/stable/regression/rse.html
